types.go

```diff
@@ -176,6 +176,12 @@ func (bs *BundleStatus) ShortString() string {
 			buf.WriteString(cond.Reason)
 			buf.WriteByte('"')
 		}
+		if cond.Message != "" {
+			buf.WriteByte(' ')
```

Comment:
Isn't it better to write the result of `fmt.Sprintf()`? It would make it more readable